index.html

```diff
@@ -1,146 +1,150 @@
 <!DOCTYPE html>
 <html lang="pt-br">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./style/style.css" />
-    <link rel="icon" href="./assets/image 1 (1).png" />
-    <title>SOS</title>
-  </head>
 
-  <body>
-    <header>
-      <a href="./mobile/index.html">
-        <h1 class="headerTitle">SOS</h1>
-      </a>
-      <nav>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined" rel="stylesheet" />
+  <link rel="stylesheet" href="./style/style.css" />
+  <link rel="icon" href="./assets/image 1 (1).png" />

```

Comment:
Criar um padrão de nomenclatura para seus ícones de forma intuitiva

exemplo: caso outro Dev for realizar manutenção no seu código ele precisa saber qual o ícone apenas lendo seu nome 

ex:
logoFavIcon.svg